dora/src/bytecode/generator.rs

```diff
@@ -488,6 +488,53 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
         }
     }
 
+    fn visit_expr_delegation(&mut self, expr: &ExprDelegationType, dest: DataDest) -> Register {
+        let call_type = self.src.map_calls.get(expr.id).unwrap().clone();
+        let fct_id = call_type.fct_id().unwrap();
```

Comment:
Nit: I don't think we need `fct_id` here.

Reply:
How would this work?

Reply:
How would this work exactly?

Reply:
fct_id and callee_id are the same, so one of the variables is not needed.